app/views/coronavirus_landing_page/_page_header.html.erb

```diff
@@ -30,25 +30,48 @@
       text: "Announcements",
       padding: true,
       font_size: 19,
-      heading_level: 2
+      inverse: true
     } %>
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-one-third">
-        <h3 class="gem-c-heading  gem-c-heading--font-size-19 gem-c-heading--padding gem-c-heading--border-top-2">
-          <a href="https://www.gov.uk/government/publications/closure-of-educational-settings-information-for-parents-and-carers/closure-of-educational-settings-information-for-parents-and-carers">
+        <% link = capture do %>
+          <a href="https://www.gov.uk/government/publications/closure-of-educational-settings-information-for-parents-and-carers/closure-of-educational-settings-information-for-parents-and-carers" class="govuk-link covid__page-header-link">
             Schools and childcare are only open to children of key workers
```

Comment:
Should we not have all of this in the content yaml file:

https://github.com/alphagov/collections/blob/master/config/locales/en/coronavirus_landing_page.yml

Reply:
Yes, good point, will have a look.

Reply:
Leave it for now, I'll pick it up after the latest update